addons/payment/views/payment_templates.xml

```diff
@@ -56,14 +56,79 @@
+                                      class="badge rounded-pill text-bg-secondary ms-1">
+                                        + <t t-esc="fees_by_provider.get(token.provider_id)"
+                                             t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                        Fees
+                                </span>
+                                <!-- === "Test token" badge === -->
+                                <t t-call="payment.test_token_badge"/>
+                                <!-- === "Unpublished" badge === -->
+                                <span t-if="not token.provider_id.is_published" class="badge rounded-pill text-bg-danger ms-1">
+                                    Unpublished
+                                </span>
+                                <div class="ms-4">
+                                    <span class="fst-italic small text-muted">
+                                        Managed by
+                                        <!-- It would not work with custom payment provider e.g. 
+                                          "Wire transfer" as it would just show "custom", but
+                                          for now we don't tokenize custom providers, so it's
+                                          not a problem -->
+                                        <span class="token_payment_provider_code"
+                                              t-esc="dict(provider._fields['code']._description_selection(provider.env))[provider.code]"/>
+                                        - created on 
+                                        <span class="token_payment_creation_date" t-esc="token.create_date.date()"/>
+                                    </span>
+                                </div>
+                            </label>
+                        </div>
+                        <!-- === Token inline form === -->
+                        <div t-attf-id="o_payment_token_inline_form_{{token.id}}"
+                             name="o_payment_inline_form"
+                             class="card-footer d-none">
+                            <t t-set="token_inline_form_xml_id"
+                               t-value="token.sudo().provider_id.token_inline_form_view_id.xml_id"/>
+                            <!-- === Inline form content (filled by provider) === -->
+                            <div t-if="token_inline_form_xml_id" class="clearfix">
+                                <t t-call="{{token_inline_form_xml_id}}">
+                                    <t t-set="token" t-value="token"/>
+                                </t>
+                            </div>
+                        </div>
+                    </t>
+                </div>
+            </t>
+            <!-- === Providers === -->
+            <div class="mt-3 fs-4" t-if="token_count>0"> 
```

Comment:
Swap the `class` and the `t-if`; the code structure comes before the element style. Also, some previous comments apply here too.